### PR TITLE
Further copy updates

### PR DESCRIPTION
**Pull request by marialani**
## What does this change?
Updates to the copy on the "Our organisation" page

### CODE REVIEW

File: src/pages/our-organisation/index.tsx [hunk 1]
```diff
@@ -218,11 +221,11 @@ const HomePage = () => (
...
           <p>
-            The majority of our staff, around 90% of our workforce, are employed
-            in the UK, and we have growing operations in the US and Australia.
-            We value and respect all differences in all people, seen and unseen,
-            and aspire to an open, supportive and inclusive culture which makes
-            the Guardian a good place to work for everyone.
+            We value and respect all differences in people, seen and unseen, and
+            aspire to an open, supportive and inclusive culture which makes the
+            Guardian a good place to work for everyone. We have a global
+            workforce, with THE MAJORITY OF our staff in the UK and growing
```
Comment (kelvin-chappell) [inline src/pages/our-organisation/index.tsx:227]:
```suggestion
            workforce, with the majority of our staff in the UK and growing
```

File: src/pages/our-organisation/index.tsx [hunk 2]
```diff
@@ -253,8 +256,10 @@ const HomePage = () => (
           <p>
             GMG operates a diverse revenue model including reader revenues,
             advertising, jobs advertising, licensing and philanthropic funding.
-            Revenue from readers now accounts for over 50% of GMG's annual
-            revenues.
+            We are increasingly global with growing commercial operations in the
+            US and Australia. Revenue from readers now accounts for over 50% of
+            GMG's annual revenue, with more than half of reader revenue coming
+            from readers outside the UK
```
Comment (kelvin-chappell) [inline src/pages/our-organisation/index.tsx:262]:
```suggestion
            from readers outside the UK.
```

File: src/pages/our-organisation/index.tsx (hunk 1 above)
Comment (marialani) [reply src/pages/our-organisation/index.tsx:227]: Think they've gone with caps here for emphasis

File: src/pages/our-organisation/index.tsx [hunk 3]
```diff
@@ -193,7 +193,10 @@ const HomePage = () => (
             <p>
               In 2019, we became the first major news organisation to certify as
               a B Corporation, and made a climate pledge to our readers
-              committing to reach net zero carbon emissions by 2030.
+              committing to reach net zero carbon emissions by 2030. In 2020 we
```
Comment (rBangay) [inline src/pages/our-organisation/index.tsx:196]: The "In 2020" part should be in a new paragraph :)

**Review by rBangay — APPROVED** [review]

**Pull request merged**